Cargo.toml

```diff
@@ -13,13 +13,15 @@ exclude = ["data"]
 resolver = "2"
 
 [features]
-default = ["recovery", "log-trace"]
+default = ["recovery", "log-trace", "crossbeam-channel"]
 recovery = ["sysinfo"]
 log-trace = []  # test only 
 log-debug = []  # test only
+crossbeam-channel = ["notify/default"]
+no-crossbeam-channel = ["notify/macos_kqueue"]
```

Comment:
So, what has made me a bit uneasy was this: every time we compile for a MacOS target we would have to use this `no-crossbeam-feature`? Could we use `[target.'cfg(target_os = "macos")'.dependencies]` instead? (I have googled and it says this is a bit broken, but at least worth a shot. Else, could we use. `cfg` in the code directly?)